ibm/service/contextbasedrestrictions/resource_ibm_cbr_zone.go

```diff
@@ -344,46 +309,28 @@ func resourceIBMCbrZoneCreate(context context.Context, d *schema.ResourceData, m
 		return diag.FromErr(fmt.Errorf("CreateZoneWithContext failed %s\n%s", err, response))
 	}
 
-	// handle Eventual consistency case
-	err = readNewCbrZone(contextBasedRestrictionsClient, context, *zone.ID)
-	if err != nil {
-		return diag.FromErr(err)
-	}
-
 	d.SetId(*zone.ID)
-	return resourceIBMCbrZoneRead(context, d, meta)
-}
 
-func resourceIBMCbrZoneRead(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	contextBasedRestrictionsClient, err := meta.(conns.ClientSession).ContextBasedRestrictionsV1()
-	if err != nil {
-		return diag.FromErr(err)
-	}
-
-	var zone *contextbasedrestrictionsv1.Zone
-	var version string
-	var found bool
-	zone, version, found, err = getZone(contextBasedRestrictionsClient, context, d.Id())
-	if err != nil {
-		return diag.FromErr(err)
-	}
-	if !found {
-		d.SetId("")
-		return nil
+	version := response.Headers.Get("Etag")
+	if fromErr := ResourceIBMCbrZoneSetData(zone, version, d); fromErr != nil {
+		return diag.FromErr(fmt.Errorf("Error setting zone's resource data: %s", fromErr))
 	}
+	return nil
+}
 
-	if err = d.Set("name", zone.Name); err != nil {
+func ResourceIBMCbrZoneSetData(zone *contextbasedrestrictionsv1.Zone, version string, d *schema.ResourceData) diag.Diagnostics {
```

Comment:
Is there a reason why the zone resource does not have transaction ID and correlation ID information?

Reply:
looks like transaction ID and correlation ID are removed from here: the https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5505/files#diff-6b5bf268286a585272f1b259bf479cb72deffd67282020b1a8e1bc83b7df1edaR368

Reply:
I still see them defined [here](https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5505/files#diff-a2b3b9ab5d60a8d750a60903e0f3e06131ca3ac7c5586eea215677cae41c07e9R97-R108).

Reply:
yes, they are still in `resource_ibm_cbr_zone_addresses.go`, but missing from `resource_ibm_cbr_zone.go`